README.md

```diff
@@ -21,27 +21,27 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+That is a migration file and the numbers at the beginning are the timestamp.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+the router checks the controller for routes to views
```

Comment:
A controller action will probably result in some view being generated. But the router isn't concerned with the views -- all it's doing is determining which controller action to trigger based on the request that comes in from the browser.